e2e/nomostest/new.go

```diff
@@ -286,20 +286,28 @@ func FreshTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 		// a single test.
 		connectToLocalRegistry(nt)
 	}
-	if !opts.IsEphemeralCluster {
-		// We aren't using an ephemeral cluster, so make sure the cluster is
-		// cleaned before and after running the test.
+	t.Cleanup(func() {
+		if *e2e.DestroyClusters {
+			nt.T.Logf("Skipping cleanup because cluster will be destroyed")
```

Comment:
Maybe add (--destroy-clusters=true) here as well by way of explaination

Reply:
agreed, done